cmd/config.go

```diff
@@ -309,17 +310,30 @@ func getConsolidatedConfig(fs afero.Fs, cliConf Config, runner lib.Runner) (conf
 	conf = conf.Apply(envConf).Apply(cliConf)
 	conf = applyDefault(conf)
 
+	// TODO(imiric): Move this validation where it makes sense in the configuration
+	// refactor of #883. Currently we validate CLI flags in cmd.getOptions. Yet
+	// there is no place where configuration is validated after applying all other
+	// sources, like environment variables (besides inline in the runCmd itself...).
```

Comment:
> Yet there is no place where configuration is validated after applying all other sources

I don't think that's the case - see `validateConfig()` at the bottom of this file. It's called as a part of `deriveAndValidateConfig()` in https://github.com/loadimpact/k6/blob/91e144edd39ec9b82d69c4d9c5c7d93a81fda01e/cmd/run.go#L174

that happens well after all config options have been consolidated. In your case, since `SummaryTrendStats` is actually part of `lib.Options`, you can add this simple validation in the currently very anemic `lib.Options.Validate()`: https://github.com/loadimpact/k6/blob/91e144edd39ec9b82d69c4d9c5c7d93a81fda01e/lib/options.go#L435-L440

Reply:
@na-- True, I forgot about `validateConfig()`. I think I initially did the validation in `Options.Validate()`, but don't remember why it was moved. Or maybe that was for another PR...

In any case, I tried this now, but calling `ui.ValidateSummary()` from `lib.Options.Validate()` introduces circular imports. :man_facepalming: And moving `ValidateSummary` (along with `validatePercentile()` to `lib` would also require moving all global errors defined in `ui/summary.go` if we want to avoid the circular import...

So not sure, this is messy, as validation functions shouldn't be in `ui` anyway, but I don't see a clear way of improving it before #883.

I updated the comment in 8bbc0272 to remove the "there is no place" line. Let me know.

Reply:
> So not sure, this is messy, as validation functions shouldn't be in `ui` anyway, but I don't see a clear way of improving it before #883.

hmm yeah, it's far from ideal, but it's not worth doing a huge refactoring just for this now, we can fix it in #883 

but still, at least move `ui.ValidateSummary()` out of `getConsolidatedConfig()` and in `validateConfig()` - it's in the same package, so there shouldn't be any conflicts

Reply:
nevermind, that needs to be fixed in #1007 since that function just returns a warning now